packages/compartment-mapper/src/node-powers.js

```diff
@@ -138,10 +144,10 @@ const makeReadPowersSloppy = ({
         const realPath = await fs.promises.realpath(
           fileURLToPath(location).replace(/\/$/, ''),
         );
-        return `${pathToFileURL(realPath)}/`;
+        return `${pathToFileURL(realPath).href}/`;
       } else {
         const realPath = await fs.promises.realpath(fileURLToPath(location));
-        return pathToFileURL(realPath).toString();
+        return pathToFileURL(realPath).href;
```

Comment:
So I am curious about this.  I know that we (who?) settled on using `.href` to convert a `URL to a `string`.

`URL.prototype.toString()` just returns the `href` prop.  The advantage I see to using `toString()` (or wrapping in a template literal) is that `href` is writable and might be a source of subtle bugs (e.g., accidental assignment).

Reply:
No decision has been made and this change is not necessary. I started with `toString()` but have been leaning in the direction of `href`. There isn’t a strong case either way.